security/src/test/java/org/mule/test/config/SecurityNamespaceHandlerTestCase.java

```diff
@@ -56,10 +60,10 @@ public void testSecurity() throws Exception {
 
   @Test
   public void testProvidersAreInitialized() throws Exception {
-    SecurityManager securityManager = muleContext.getSecurityManager();
-    SecurityProvider customDelegateSecurityProvider = securityManager.getProvider("initializableProvider");
-    InitTrackerSecurityProvider initTrackerSecurityProvider =
-        (InitTrackerSecurityProvider) readField(customDelegateSecurityProvider, "delegate", true);;
+    // SecurityManager securityManager = muleContext.getSecurityManager();
+    // SecurityProvider customDelegateSecurityProvider = securityManager.getProvider("initializableProvider");
+    // InitTrackerSecurityProvider initTrackerSecurityProvider =
+    // (InitTrackerSecurityProvider) readField(customDelegateSecurityProvider, "delegate", true);;
     assertThat(initTrackerSecurityProvider.isInitialised(), is(true));
```

Comment:
Shouldn't we just delete the commented code or add an issue if it's supposed to be uncommented eventually?